openfl/component/aggregator/aggregator.py

```diff
@@ -854,6 +857,7 @@ def _end_of_round_check(self):
         if self._time_to_quit():
             self.logger.info('Experiment Completed. Cleaning up...')
         else:
+            self.round_number += 1
```

Comment:
Moving the round incremement to the else statement will result in N+1 rounds being executed. This should be changed back

Reply:
No, in fact difference only in logging. Real checking that it is time-time-quit happens in `get_tasks` function. I returned it back, but we should rewrite this part of code in the future.